.github/stale.yml

```diff
@@ -0,0 +1,15 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 365
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 90
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - "Type: Feature"
+# Label to use when marking an issue as stale
+staleLabel: stale
```

Comment:
Is it possible to change this to "Status: Stale" for consistency with (most) of the other tags.